ddtrace/tracer/rules_sampler.go

```diff
@@ -78,6 +127,8 @@ type SamplingRule struct {
 	// Tags specifies the map of key-value patterns that span tags must match.
 	Tags map[string]*regexp.Regexp
 
+	Provenance Provenance
```

Comment:
```suggestion
	provenance provenance
```

As said in my previous comment, I don't feel we need to export this field. Do we expect users to set this value for any reason? If I understand correctly, the zero value for `provenance` would be a sensible default as long as it's associated to `local` semantics.

Reply:
done.